locales/es-ES/messages.po

```diff
@@ -605,6 +641,15 @@ msgstr ""
 msgid "Yes"
 msgstr ""
 
+msgid "You have %s item(s) marked for review."
+msgstr ""
+
+msgid "You have %s unanswered question(s) and have %s item(s) marked for review."
+msgstr ""
```

Comment:
Tiene %s pregunta(s) sin respuesta y %s ítem(s) marcado(s) para revisión. 

This precisly the translation that is concerned by the task